NEWS.md

```diff
@@ -121,6 +121,9 @@ core developer team.
 
 * `sec_axis()` now accepts functions as well as formulas (@yutannihilation, #3031).
 
+* `Stat$compute_layer()` and `Position$compute_layer()` now have the same names of
+  arguments (@yutannihilation, #3202).
```

Comment:
I feel something is missing here. "have now the same names of arguments *as other `compute_layer()` functions*"?

Reply:
Thanks. I should probably say something like "The arguments of `Stat*$compute_layer()` and `Position*$compute_layer()` are now renamed to match the ones of `Stat$compute_layer()` and `Position$compute_layer()`." Does this make sense...?

Reply:
Yes, maybe with this edit: "...now renamed to *always* match the ones...". After all, they mostly were matching already, but not always.

Reply:
Got it!